### PR TITLE
added blog

### PR DESCRIPTION
**Pull request by JasminFarhi**
added blog link and image to site

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,8 +94,11 @@ <h2 class="to-animate">Legal</h2>
       <li><a href="https://www.facebook.com/profile.php?id=100071134837958" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
       <li><a href="https://www.instagram.com/dltx.io/" target="_blank"><i class="fab fa-instagram"></i></a></li>
       <li><a href="https://www.linkedin.com/company/dltx/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></i></i></i></a></li>
       <li><a href="https://github.com/dltxio" target="_blank"><i class="fab fa-github-square"></i></a></li>
       <li><a href="https://discord.gg/kAW8yvMgJN" target="_blank"><i class="fab fa-discord"></i></a></li>
+      
+
```
Comment (Mitchell139) [inline index.html:101]:
```suggestion
```

**Review by Mitchell139 — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

File: careers.html [hunk 2]
```diff
@@ -86,6 +86,7 @@ <h3>We will soon be looking for business analysts and business development roles
       <li><a href="https://www.facebook.com/profile.php?id=100071134837958" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
       <li><a href="https://www.instagram.com/dltx.io/" target="_blank"><i class="fab fa-instagram"></i></a></li>
       <li><a href="https://www.linkedin.com/company/dltx/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></i></i></i></a></li>
```
Comment (bitcoinbrisbane) [inline careers.html:89]: Seems too many closing tags?

**Review by bitcoinbrisbane — COMMENTED** [review]
One small thing

File: index.html [hunk 3]
```diff
@@ -94,6 +94,7 @@ <h2 class="to-animate">Legal</h2>
       <li><a href="https://www.facebook.com/profile.php?id=100071134837958" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
       <li><a href="https://www.instagram.com/dltx.io/" target="_blank"><i class="fab fa-instagram"></i></a></li>
       <li><a href="https://www.linkedin.com/company/dltx/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></i></i></i></a></li>
```
Comment (bitcoinbrisbane) [inline index.html:97]:
```suggestion
      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></a></li>
```

File: services.html [hunk 4]
```diff
@@ -106,6 +106,7 @@ <h2 class="to-animate">Our Clients</h2>
       <li><a href="https://www.facebook.com/profile.php?id=100071134837958" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
       <li><a href="https://www.instagram.com/dltx.io/" target="_blank"><i class="fab fa-instagram"></i></a></li>
       <li><a href="https://www.linkedin.com/company/dltx/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></i></i></i></a></li>
```
Comment (bitcoinbrisbane) [inline services.html:109]:
```suggestion
      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></a></li>
```

File: trial-day.html [hunk 5]
```diff
@@ -73,6 +73,7 @@ <h3 class="to-animate">What happens after my Trial Day?</h3>
       <li><a href="https://www.facebook.com/profile.php?id=100071134837958" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
       <li><a href="https://www.instagram.com/dltx.io/" target="_blank"><i class="fab fa-instagram"></i></a></li>
       <li><a href="https://www.linkedin.com/company/dltx/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></i></i></i></a></li>
```
Comment (bitcoinbrisbane) [inline trial-day.html:76]:
```suggestion
      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></a></li>
```

File: venture-studio.html [hunk 6]
```diff
@@ -75,6 +75,7 @@ <h3 class="to-animate">Block 3</h3>
       <li><a href="https://www.facebook.com/profile.php?id=100071134837958" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
       <li><a href="https://www.instagram.com/dltx.io/" target="_blank"><i class="fab fa-instagram"></i></a></li>
       <li><a href="https://www.linkedin.com/company/dltx/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
+      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></i></i></i></a></li>
```
Comment (bitcoinbrisbane) [inline venture-studio.html:78]:
```suggestion
      <li><a href="https://medium.com/@dltx.io" target="_blank"><i class="fas fa-pen-square"></i></a></li>
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Seems some tag issues

**Pull request merged**